src/screens/VisualTests/SnapshotComparison.stories.tsx

```diff
@@ -122,6 +122,13 @@ export const FirstPassed: Story = {
       })
     ),
   },
+  play: playAll(async ({ canvasElement, canvasIndex }) => {
+    const canvas = within(canvasElement);
+    const menu = await canvas.findByRole("button", { name: "Safari" });
+    await userEvent.click(menu);
+    const items = await screen.findAllByText("Chrome");
+    await userEvent.click(items[canvasIndex]);
+  }),
```

Comment:
I added this `play` function since `useTests` now selects the first non-passed test/comparison by default. But, this story's comment kind of sounds like we expected to show the first passed test by default. Since that is no longer the case, is this story still valid at all?

Reply:
Let's drop the `play` function, instead update the code comment above and update the snapshot.